src/fmu/dataio/aggregation.py

```diff
@@ -216,8 +218,8 @@ def _generate_aggrd_metadata(
         if not self.operation:
             raise ValueError("The 'operation' key has no value")
 
-        # use first as template but filter away invalid entries first:
-        template = _utils.filter_validate_metadata(self.source_metadata[0])
+        # use first as template
+        template = copy.deepcopy(self.source_metadata[0])
```

Comment:
As far as I could see from the doc, Pydantic should default to "Ignore" on unknown fields so it should be okay to remove this filtering method. Is this correct?

Reply:
Yes, plus we expect that incoming metadata is already valid and will be further validated once we put it into the Pydantic model